lang/en.json

```diff
@@ -95,6 +95,7 @@
   "CHAT.DURATION": "Duration",
   "CHAT.FOCUS_APPLIED" : "applied to highlighted dice",
   "CHAT.ARMOUR_SUBTRACTED" : "after armour of",
+  "CHAT.APPLY_DAMAGE" : "apply damage to target",
```

Comment:
Can you capitalize this to look cleaner? 
![image](https://user-images.githubusercontent.com/28637157/122830600-30373900-d2ae-11eb-8fe3-4e2aae8bb357.png)

Reply:
Certainly